deps/tools/common.mk

```diff
@@ -174,7 +174,6 @@ UNINSTALL_$(strip $1) := $2 staged-uninstaller
 
 $$(build_prefix)/manifest/$(strip $1): $$(build_staging)/$2.tgz | $(build_prefix)/manifest
 	-+[ ! -e $$@ ] || $$(MAKE) uninstall-$(strip $1)
-	mkdir -p $$(build_prefix)
 	$(UNTAR) $$< -C $$(build_prefix)
```

Comment:
don't we need to do the same here as well?

Reply:
No these files aren't built by Tar.jl, so they don't share the same issue with unpacking them